test/unit/mailer_test.rb

```diff
@@ -23,7 +24,28 @@ class MailerTest < ActiveSupport::TestCase
     end
   end
 
+  context "sending mail for mfa required soon announcement" do
+    setup do
+      @user = create(:user, mfa_level: "disabled")
+      create(:rubygem, owners: [@user], downloads: MIN_DOWNLOADS_FOR_MFA_REQUIRED_POLICY)
+
+      Gemcutter::Application.load_tasks
+      Rake::Task["mfa_policy:reminder_enable_mfa"].invoke
+      Delayed::Worker.new.work_off
+    end
+
+    should "send mail to users" do
+      refute_empty ActionMailer::Base.deliveries
```

Comment:
This is empty 🤔 . Need to debug the test. Will wrap up later today 💪

Reply:
👍 I think the test is failing because you're querying for [greater than 180million downloads](https://github.com/rubygems/rubygems.org/pull/3166/files#diff-ca4ffd6e42af383a715c6c72f563969227bc4c41dc40907a0a90a25f763dabccR42-R43) and not greater and equal to like in the last email blast. Alt: if you add one to the amount of downloads it should work as well.

Reply:
Ha 🙈! Yea! I realized that today too. Thanks Jenny 😅.